lib/ImageResize.php

```diff
@@ -249,16 +249,6 @@ public function save($filename, $image_type = null, $quality = null, $permission
             if (version_compare(PHP_VERSION, '5.5.0', '<')) {
                 throw new ImageResizeException('For WebP support PHP >= 5.5.0 is required');
             }
-            if( !empty($exact_size) && is_array($exact_size) ){
-                $dest_image = imagecreatetruecolor($exact_size[0], $exact_size[1]);
-                $background = imagecolorallocate($dest_image, 255, 255, 255);
-                imagefilledrectangle($dest_image, 0, 0, $exact_size[0], $exact_size[1], $background);
-            } else{
-                $dest_image = imagecreatetruecolor($this->getDestWidth(), $this->getDestHeight());
-                $background = imagecolorallocate($dest_image, 255, 255, 255);
-                imagefilledrectangle($dest_image, 0, 0, $this->getDestWidth(), $this->getDestHeight(), $background);
```

Comment:
Do you have any test case about this `webp` transparency?

Reply:
Sure, here you are : https://we.tl/t-KAjNhXovT6

The original png + the webp generated with the code modified above

Reply:
Hi @wearemetabolism, thank you for your reply.

Could you write some test cases with PHPUnit on this PR?

I don't think it's a proper way to provide the project archive about this test I mention...

Reply:
Hi @peter279k sure, I've added some test, I don't understand why travis failed.
Tests are ok by my side : OK (42 tests, 76 assertions)

Reply:
Hi @wearemetabolism, thank you for your reply.

It looks strange after tracing the Travis CI build.

I will take a look at this log clearly.

BTW, could you rebase the commits to one commit so that we can looks the changes easily.